app/main/views/index.py

```diff
@@ -19,6 +19,17 @@
 from app.main import main
 from app.utils import assess_contact_type, bytes_to_pretty_file_size
 
+FILE_EXTENSION_TO_PRETTY_FILE_TYPE = {
+    'csv': 'CSV',
```

Comment:
Maybe we should add the word `file` on to this one? I think I'd always say "CSV file" rather than just "CSV" ... but possibly I'm wrong / it's just my opinion.

Reply:
I've always seen it as a CSV file (and I'm not techy at all) so would be inclined to agree with @samuelhwilliams

Reply:
Thanks. I will update `CSV` to be `CSV file`